src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Deployment/CreateOrderStatuses.sql

```diff
@@ -0,0 +1,10 @@
+﻿DECLARE @submitted AS nvarchar(30) = N'Submitted';
+DECLARE @unsubmitted AS nvarchar(30) = N'Unsubmitted';
+
+IF NOT EXISTS (
+  SELECT *
+  FROM dbo.OrderStatus
+  WHERE [Name] IN (@submitted, @unsubmitted))
+BEGIN
+	INSERT INTO dbo.OrderStatus ([Name]) VALUES (@submitted), (@unsubmitted)

```

Comment:
What are the Id's in this case. Would you need to specify them in the insert?

Reply:
Added an auto increment on them, they are INT starting from 1 ... n

Reply:
👍